src/popup/components/AddressBook/AddressBookList.vue

```diff
@@ -63,19 +72,24 @@ export default defineComponent({
   },
   props: {
     modelValue: Boolean,
+    /** Whether the list is being used in an account selector or not  */
+    isSelector: Boolean,
   },
-  emits: ['update:modelValue'],
+  emits: ['update:hide-buttons', 'selectAddress'],
```

Comment:
I suggest to maintain one naming case for events. I suggest:
```suggestion
  emits: ['update:hideButtons', 'select-address'],
```